pkg/scheduler/framework/plugins/nodeunschedulable/node_unschedulable.go

```diff
@@ -25,8 +25,8 @@ import (
 	framework "k8s.io/kubernetes/pkg/scheduler/framework/v1alpha1"
 )
 
-// NodeUnschedulable is a plugin that priorities nodes according to the node annotation
-// "scheduler.alpha.kubernetes.io/preferAvoidPods".
+// NodeUnschedulable is a plugin that checks if a pod can be scheduled on a
```

Comment:
nit: a -> an :)

Reply:
Thanks for your reminding, I would change that after we agree on the comment